services/data-info/src/data_info/routes/avus.clj

```diff
@@ -13,25 +14,30 @@
     (GET* "/avus" [:as {uri :uri}]
       :query [{:keys [user]} StandardUserQueryParams]
       :return AVUGetResult
+      :middlewares [wrap-metadata-base-url]
       :summary "List AVUs (administrative)"
-      :description (str "List iRODS AVUs associated with a data item. Include administrative/system AVUs."
-(get-error-code-block "ERR_NOT_A_USER, ERR_NOT_READABLE"))
+      :description
+          (str "List all AVUs associated with a data item. Include administrative/system iRODS AVUs."
```

Comment:
Hm, it might do fine in aggregate with the other endpoints. There's documentation of the exact return values anyway, so I think I might have been more confused by it in the PR than I would be in reality.